index.js

```diff
@@ -22,6 +22,12 @@ const {
   specifiedRules,
   execute
 } = require('graphql')
+const {
+  mergeResolvers,
+  makeExecutableSchema,
+  printSchemaWithDirectives,
+  getResolversFromSchema
+} = require('graphql-tools')
```

Comment:
I would really prefer not adding a dependency to graphql-tools. This is likely doing a lot of work, and it might be way too "magical" to maintain long term. Can we make this simple for users to do the same outside of mercurius?

Reply:
I think that's reasonable. I will have to do more research on the subject. Primarily, I'll attempt to look into implementing the custom directives relying only on `graphql-js`. Meanwhile, this PR could be closed/rejected, and I could open a new one at a later time.